vectorMath.py

```diff
@@ -0,0 +1,79 @@
+import math
+
+
+def magnitude(u):
+    index = 0
+    sum = 0
+    for i in u:
+        mag = (u[index])**2
+        index += 1
+        sum = sum + mag
+    return math.sqrt(sum)
+
+
+def dotProduct(u, v):
+    index = 0
+    sum = 0
+    for i in u:
+        dot = (u[index] * v[index])
+        sum = sum + dot
+        index += 1
+    return sum
+
+
+def angleBtwnVectors(u, v):
+    theta = math.acos((dotProduct(u, v))/((magnitude(u))*(magnitude(v))))
+    return math.degrees(theta)
+
+
+def crossProduct(u, v):
+    cross = [(u[1]*v[2]-v[1]*u[2]),
+             (v[0]*u[2]-u[0]*v[2]),
+             (u[0]*v[1]-u[1]*v[0])]
+    return cross
+
+
+choice = raw_input('You can choose:\n'
```

Comment:
You keep trying to make your scripts interactive. There are more conventional ways to do this.
1: Python already has an interactive shell. If this were a library, couldn't you just import it and interact with it like that?
2: You could make it a CLI (command line interface) just like cd, mv, ls. Then, you could you could call it from the command line.
3: You could make a legit UI. Remember flask?

Reply:
That would be cool. I don't know anything about interactive stuff right now